notebooks/signac_101_Getting_Started.ipynb

```diff
@@ -156,9 +166,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/Users/atravitz/software/signac-docs/docs/source/signac-examples/notebooks/projects/tutorial\n",
```

Comment:
Hmm. We may want to re-do some of the content so that your personal user paths don't show up, @atravitz. There are considerations for both reproducibility and privacy. I don't know what to recommend here yet, but absolute paths show up in quite a few notebooks. We ought to address this before merging.

Reply:
Fixed in #25.